lnrpc/lightning.proto

```diff
@@ -1342,6 +1342,13 @@ enum CommitmentType {
     channel before its maturity date.
     */
     SCRIPT_ENFORCED_LEASE = 4;
+
+    /*
+    A channel that uses musig2 for the funding output, and the new tapscript
+    features where relevant.
+    */
+    // TODO(roasbeef): need script enforce mirror type for the above as well?
```

Comment:
for my understanding, what is a "mirror type"?

Reply:
So one for the script enforced lease scheme we use in Pool.